courseIntro.js

```diff
@@ -313,6 +312,7 @@ class courseIntro extends Phaser.Scene {
         //this.congrats.alpha = 0;
 	    this.returnDoor.alpha = 1;
         this.coin0.alpha = 1.0;
+        if(roomProgress > 1000) this.coin0.alpha = 0.0;
```

Comment:
Why set it to 1 and then maybe set it to 0? This might cause a "flicker"? After these 2 statements, it needs to be either 1 or 0 depending on the roomProgress so use an if-else statement.